android/src/main/java/ee/forgr/biometric/NativeBiometric.java

```diff
@@ -68,44 +68,15 @@ public class NativeBiometric extends Plugin {
 
   private SharedPreferences encryptedSharedPreferences;
 
+  private static final boolean allowDeviceCredentials = true;
   private int getAvailableFeature() {
-    // default to none
-    int type = NONE;
-
-    // if has fingerprint
-    if (
-      getContext()
-        .getPackageManager()
-        .hasSystemFeature(PackageManager.FEATURE_FINGERPRINT)
-    ) {
-      type = FINGERPRINT;
+    if (allowDeviceCredentials && !isCurrentSDK29OrEarlier()) {
```

Comment:
This change most likely breaks something in the JS interface, let's not completely change the function response here.